src/components/Video/VideoProperties.vue

```diff
@@ -1,12 +1,31 @@
 <template>
   <div>
     <b-form>
-      <b-form-group
-        label="Processing:"
-        horizontal>
-        <b-input
-          :value="processingState"
-          disabled />
+      <b-form-group>
+        <h3>Properties</h3>
+        <div
+          v-for="prop of properties"
+          :key="prop.key">
+          <p v-if="recording.batteryLevel && prop.key === 'batteryLevel'">
+            <strong>Battery Level: </strong><BatteryLevel :battery-level="recording.batteryLevel"/>
+          </p>
+          <p v-else-if="recording.location && prop.key === 'location'">
+            <strong>Location: </strong>{{ parseLocation }}
+          </p>
+          <p v-else-if="recording.additionalMetadata && prop.key === 'additionalMetadata'">
+            <strong>Additional Metadata:</strong>
+            <ul>
+              <li
+                v-for="(key, index) of Object.keys(recording['additionalMetadata'])"
+                :key="index">
+                {{ key }}: {{ recording['additionalMetadata'][key] }}
+              </li>
+            </ul>
```

Comment:
This is fine but I wonder if it would look better as a two column table. Your call though.

Reply:
I've implemented that - see what you think